connect/src/spirc.rs

```diff
@@ -873,8 +874,6 @@ impl SpircTask {
         }
 
         match update.typ() {
-            MessageType::kMessageTypeHello => self.notify(Some(ident)),
```

Comment:
Why remove this?

Reply:
Because it caused recursion when 2 librespot instances were on the same network. And why were echoing a Hello from another client as that client in the 1st place?

Reply:
The 2 instances start to just continually notify back and forth forever as fast as they can.

Reply:
I added that back. Although I'm not sure why we're answering a hello by notifying with the ident of the sender of the message?

Reply:
I'll come back to all that when I fix discovery.

Reply:
If the current behavior is correct we should probably keep track of other clients so we can be like "Yep, we already responded to that Hello, we don't need to notify over and over again forever."

Reply:
I honestly don't know if the behavior is correct, that's why I was asking. If you've found this fixes something then good but I start from the assumption that it served some purpose.

If this is indeed a fix then it would be cleaner to have it as a separate PR I think for reasons of traceability in the future.

Reply:
Looking at it since discovery does not work for me I don't really know how it behaves without directly logging in so it's probability best to leave it in until I fix discovery.